test/e2e/pkg/grammar/abci_grammar.md

```diff
@@ -1,15 +1,17 @@
 ```
-package "github.com/cometbft/cometbft/test/e2e/pkg/grammar/clean-start/grammar-auto"
+package "github.com/cometbft/cometbft/test/e2e/pkg/grammar/grammar-auto"
 
-Start : CleanStart ;
+Start : CleanStart | Recovery;
 
-CleanStart : InitChain StateSync ConsensusExec | InitChain ConsensusExec ;
+CleanStart : InitChain ConsensusExec | StateSync ConsensusExec ;
```

Comment:
My understanding of the grammar changes is that, now `InitChain` is always preceded by `Info`, is that the case here?

Oh wait... is it because we're not checking `Info` in the tests? So no point in adding it?

Reply:
Yes, two things are new:
1. the previous grammar allowed both InitChain and StateSync. Now we know this is not correct, so we have InitChain or StateSync
2. Info is missing in the grammar. We do not check it because it can happen at any point.